client/components/gravatar-with-hovercards/hovercard-content.jsx

```diff
@@ -106,18 +140,69 @@ function HovercardContent( props ) {
 										className="gravatar-hovercard__primary-blog-card-follow-button"
 										siteUrl={ primaryBlogUrl }
 										hasButtonStyle
-										followSource="gravatar-hovercard"
+										followSource="gravatar-hovercard__primary-blog-card"
 									/>
 								</div>
 							) }
 						</div>
 
 						<div className="gravatar-hovercard__footer">
-							{ /* TODO: Add recommended blogs list */ }
+							{ shouldShowRecommendedBlogs && (
+								<div className="gravatar-hovercard__recommended-blogs">
+									<div className="gravatar-hovercard__recommended-blogs-header">
+										<h5 className="gravatar-hovercard__recommended-blogs-title">
+											{ translate( 'Recommended blogs' ) }
+										</h5>
+										<a
+											className="gravatar-hovercard__recommended-blogs-view-all"
+											href={ recommendedBlogsLink }
+											onClick={ ( e ) => {
+												e.preventDefault();
+												page( recommendedBlogsLink );
+											} }
+										>
+											{ translate( 'View all' ) }
+										</a>
+									</div>
+									<ul className="gravatar-hovercard__recommended-blogs-list">
+										{ shuffle( recommendedBlogs )
+											.slice( 0, 3 )
+											.map( ( blog ) => {
+												const { image, name, feed_URL: feedUrl } = blog.meta?.data?.feed || {};
+
+												// The default feed image for sites with no icon is
+												// awful for this case, treat it as no image to fallback
+												// to the globe icon.
+												const siteIcon = image.includes( 's0.wp.com/i/buttonw-com.png' )
+													? null
+													: image;
```

Comment:
I'm getting `Uncaught TypeError: Cannot read properties of undefined (reading 'includes')` for my list. It's probably because I'm testing with test sites that are missing this metadata. It looks like adding an `image &&` check will fix it:

```suggestion
													const siteIcon =
													image &&
													typeof image === 'string' &&
													image.includes( 's0.wp.com/i/buttonw-com.png' )
														? null
														: image;
```

Reply:
Ah good catch. I think `image?.includes` should probably be enough of a check in this case?

Reply:
That fixed it for me, thanks!